src/legacy/core_plugins/console/public/np_ready/application/containers/editor/legacy/console_editor/keyboard_shortcuts.ts

```diff
@@ -17,49 +17,60 @@
  * under the License.
  */
 import { throttle } from 'lodash';
+import { SenseEditor } from '../../../../models/sense_editor';
 
 interface Actions {
-  input: any; // TODO: Wrap this in an editor interface
+  senseEditor: SenseEditor;
   sendCurrentRequestToES: () => void;
   openDocumentation: () => void;
 }
 
-export function registerCommands({ input, sendCurrentRequestToES, openDocumentation }: Actions) {
-  const throttledAutoIndent = throttle(() => input.autoIndent(), 500, {
+export function registerCommands({
+  senseEditor,
+  sendCurrentRequestToES,
+  openDocumentation,
+}: Actions) {
+  const throttledAutoIndent = throttle(() => senseEditor.autoIndent(), 500, {
     leading: true,
     trailing: true,
   });
-  input.commands.addCommand({
+  const coreEditor = senseEditor.getCoreEditor();
+
+  coreEditor.registerKeyboardShortcut({
+    keys: { win: 'Ctrl-Enter', mac: 'Command-Enter' },
     name: 'send to elasticsearch',
```

Comment:
Super nit-pick - if this is user-visible, then we should capitalize "Elasticsearch".